azure/functions/authenticationevents.py

```diff
@@ -1,6 +1,12 @@
+
 from importlib import import_module
 import json
+import jsonschema
+from jsonschema import validate
+from azure.functions import HttpRequest
```

Comment:
Once again, Any reason for this as this is native to the Http Request/Response trigger and not sure we can use this here. When debugging have you seen what you get in the converter?

Reply:
Unless you're planning planning on building this or calling the Http Response/Request code. But don't want to create inter-dependencies